app/actions/loadIcoParams.ts

```diff
@@ -0,0 +1,30 @@
+import { BigNumber } from "bignumber.js";
+import { Dispatch } from "redux";
+// import { selectAddress } from "../reducers/icoParameters";
+import { loadIcoParamsFromEnviroment } from "../utils";
+import { LOAD_ICO_PARAMS } from "./constants";
+
+export function loadIcoParamsAction(
+  startDate: string,
+  endDate: string,
+  minCap: BigNumber,
+  maxCap: BigNumber
+) {
+  return {
+    type: LOAD_ICO_PARAMS,
+    payload: {
+      startDate,
+      endDate,
+      minCap,
+      maxCap,
+    },
+  };
+}
+
+export async function loadIcoParams(dispatch: Dispatch<any>, getState: any) {
+  // const address = selectAddress(getState().icoParameters);
+  // TODO :Choose when to load from ENV or Smart_Contract
+  // const { minCap, maxCap, startDate, endDate } = await loadIcoParamsFromContract(address); // FROM SMART CONTRACT
+  const { minCap, maxCap, startDate, endDate } = await loadIcoParamsFromEnviroment(); // FROM ENV
+  dispatch(loadIcoParamsAction(startDate, endDate, minCap, maxCap));
+}
```

Comment:
Some of the code is commented to stay in ENV variable mode 
I think we can look into different idea to differentiate between web3 and Env for example 
- We can set two different before ICO states 
1- Before ICO ENV: with ENV start date with minumum features (only a countdown timer, no web3 ... etc)
2- Before ICO Web3: where there the application would through if connection with smart contract fials 
What do you think is the best solution and then changes can be done accordingly